cfme/tests/configure/test_access_control.py

```diff
@@ -1097,14 +1098,20 @@ def test_tenant_quota_input_validate(appliance):
 
 
 def test_delete_default_tenant(appliance):
+    """
+    Steps:
+    1. Login as an 'Administrator' user
+    2. Navigate to configuration > access control > tenants
+    3. Select default tenant('My Company') from tenants table
+    4. Delete using 'configuration > Delete selected items'
+    5. Check whether default tenant is deleted or not
+    """
+    view = navigate_to(appliance.collections.tenants, "All")
     roottenant = appliance.collections.tenants.get_root_tenant()
-    view = navigate_to(appliance.collections.tenants, 'All')
-    for row in view.table.rows():
-        if row.name.text == roottenant.name:
-            row[0].check()
     msg = 'Default Tenant "{}" can not be deleted'.format(roottenant.name)
-    with pytest.raises(Exception, match=msg):
-        view.toolbar.configuration.item_select('Delete selected items', handle_alert=True)
+    tenant = appliance.collections.tenants.instantiate(name=roottenant.name)
+    appliance.collections.tenants.delete(tenant)
+    assert view.flash.read()[0] == msg
```

Comment:
REQUIRED: 1. flash message doesn't guarantee that tenant still exists after attempt to delete it. 
please add check that tenant still exist.
2. please use view.flash.assert_message instead of your construction.